docs: :memo: add pages for current and previous decisions (#159)
seedcase-project/decisions: current.qmd

---
title: Current
listing:
  contents: why-*/*.qmd
  categories: true
  type: grid
  exclude:
    previous: true
  fields:
    - title
    - description
    - date
  sort: "date desc"
  max-items: 50
---

> These decision posts are ones we currently use and follow. All our

Reviewer: ```suggestion
These decision posts reflect the decisions we currently adhere to. All our
```
